arm-datalake-analytics/account/2016-11-01/swagger/account.json

```diff
@@ -0,0 +1,1580 @@
+        "state": {
+          "readOnly": true,
+          "type": "string",
+          "description": "the state of the Data Lake Analytics account.",
+          "enum": [
+            "active",
+            "suspended"
+          ],
+          "x-ms-enum": {
+            "name": "DataLakeAnalyticsAccountState",
+            "modelAsString": false
+          }
+        },
+        "defaultDataLakeStoreAccount": {
+          "type": "string",
+          "description": "the default data lake storage account associated with this Data Lake Analytics account."
+        },
+        "maxDegreeOfParallelism": {
+          "type": "integer",
+          "format": "int32",
+          "description": "the maximum supported degree of parallelism for this account."
+        },
+        "maxJobCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "the maximum supported jobs running under the account at the same time."
+        },
+        "dataLakeStoreAccounts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataLakeStoreAccountInfo"
+          },
+          "description": "the list of Data Lake storage accounts associated with this account."
+        },
+        "storageAccounts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageAccountInfo"
+          },
+          "description": "the list of Azure Blob storage accounts associated with this account."
+        },
+        "creationTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "the account creation time."
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "the account last modified time."
+        },
+        "endpoint": {
+          "type": "string",
+          "readOnly": true,
+          "description": "the full CName endpoint for this account."
+        }
+      },
+      "description": "The account specific properties that are associated with an underlying Data Lake Analytics account."
+    },
+    "AddDataLakeStoreParameters": {
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DataLakeStoreAccountInfoProperties",
+          "description": "the properties for the Data Lake Store account being added."
+        }
+      },
+      "description": "Additional Data Lake Store parameters."
+    },
+    "AddStorageAccountParameters": {
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/StorageAccountProperties",
+          "description": "the properties for the Azure Storage account being added."
+        }
+      },
+      "description": "Additional Azure Storage account parameters."
+    },
+    "DataLakeAnalyticsAccount": {
```

Comment:
Is this critical to do now? I ask because based on compute I would need two objects for this, too:
Resource and CreateResource, where CreateResource has "location" as a required parameter.